src/client/components/equalizer/equalizer.js

```diff
@@ -189,8 +187,38 @@ class Equalizer extends Component {
       widthCanvas,
       heightCanvas,
       trackName,
+      sound,
+      startMuteState,
     } = this.props.audioData;
 
+    const hideOrShowCanvas = sound || startMuteState ? (
+      <div>
+        <div style={{ display: 'block' }}>
+          <Graphicequaliser
```

Comment:
Please, rename this component using camel style

Reply:
done